controlplane/kubeadm/internal/webhooks/kubeadm_control_plane_test.go

```diff
@@ -333,11 +333,6 @@ func TestKubeadmControlPlaneValidateUpdate(t *testing.T) {
 					},
 				},
```

Comment:
We should make timeouts in InitConfiguration/JoinConfiguration mutable + add test coverage